lib/ssl/test/ssl_api_SUITE.erl

```diff
@@ -1464,7 +1474,7 @@ listen_socket(Config) ->
 
     {ok, _} = ssl:sockname(ListenSocket),
 
-    {error, enotconn} = ssl:send(ListenSocket, <<"data">>),
+    {error, _enotconn} = ssl:send(ListenSocket, <<"data">>),
```

Comment:
why _?
maybe code comment?

Reply:
Yeah I don't know what to do about that one, socket return a different error
and IMO we shouldn't need to be error compatible, it is easier for the user 
to get the underlying error.